.travis.yml

```diff
@@ -12,13 +12,13 @@ before_script:
   - composer install --no-interaction --prefer-source --dev
   - ~/.nvm/nvm.sh install v0.6.14
   - ~/.nvm/nvm.sh run v0.6.14
+  - '[ "$TRAVIS_PHP_VERSION" != "7.0" ] || echo "xdebug.overload_var_dump = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini'
```

Comment:
The logic here is very hard to follow. Can you update this to be the more straightforward if/then rather than if-not/else?

Shouldn't it also just set it to `0` when it's PHP 7?

```
[ "$TRAVIS_PHP_VERSION" == "7.0" ] && echo "xdebug.overload_var_dump = 0" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

/cc @jeskew